src/context/extensions/helpers/processConfig.js

```diff
@@ -209,3 +209,42 @@ function updateStateMeta(name, state, extensionConfigPaths, extensionMetaPaths,
 
     return newState.meta;
 }
+
```

Comment:
A lot of stuff going on in this module. Perhaps a candidate for more comments and maybe split/naming, as I do find it a bit hard to follow.